src/DynamoCore/Graph/Nodes/CustomNodes/Function.cs

```diff
@@ -388,8 +388,8 @@ public Symbol()
             RegisterAllPorts();
 
             ArgumentLacing = LacingStrategy.Disabled;
-
-            InputSymbol = String.Empty;
+            //TODO localize
+            InputSymbol = new TypedParameter($"DefaultInputName", "var",-1,null).ToCommentNameString();
```

Comment:
So by default all input nodes will have the same name ?
so by default all input names of a custom node will have the same name. Do you think it might be worth create a unique name ?...maybe not
Also do output nodes not have the same issue ?

Reply:
`So by default all input nodes will have the same name ?` - yes

`so by default all input names of a custom node will have the same name` - yes - but note that there should not be `inputNodes` in home workspaces.


yes, by default they will have the same name, I considered using a part of the node guid so there's a very high change the name would be unique - ie `InputName_12345` or something like that.

Something to consider is that today, they already have the same name, it's just `""`.

Output nodes do not have any validation as far as I can tell.